src/nvim/insexpand.c

```diff
@@ -3291,6 +3299,34 @@ static bool get_next_completion_match(int type, ins_compl_next_state_T *st, pos_
   return found_new_match;
 }
 
+static void get_next_buffer_token(void)
```

Comment:
Let's try to use "bufname" consistently to avoid confusion with buffer _contents_.
```suggestion
static void get_next_bufname_token(void)
```